### PR TITLE
updated output saving module following suggestions from issue #339

### PR DESCRIPTION
**Pull request by AnniekStok**
I have made updates following Tischi's suggestions here: #339

### CODE REVIEW

File: _modules/output_saving.md [hunk 1]
```diff
@@ -16,26 +16,26 @@ objectives:
...
 figure: /figures/output_saving.png
-figure_legend: Image analysis processes can yield several outputs, such as results tables, ROI sets and label masks.
+figure_legend: Image analysis processes can yield several outputs, such as measurement tables, ROI sets and label masks.
 
 activity_preface: |
-  - In the IJ Macro or Jython script, specify your output directory by pasting a string containing the path to the directory (for example r'C:\Users\UserName\Desktop' on Windows or '/Users/UserName/Desktop/' on MacOS).
-  - Open the blobs image (File > Open Samples > Blobs).
-  - Binarize it and run the 'analyze particles' command to generate different kinds of output.
-  - Save the different output types.
+  - Open the image [blob.gif](http://imagej.net/images/blobs.gif) and binarize it.
```
Comment (tischi) [inline _modules/output_saving.md:32]: @AnniekStok Could you please one of the images from our repository. In image_data there should be more than enough images with some nuclei that you could use.

**Review by tischi — COMMENTED** [review]
Perfect. Thanks a lot!
I found one more thing to fix.

File: _modules/output_saving.md (hunk 1 above)
Comment (AnniekStok) [reply _modules/output_saving.md]: @tischi, I changed it now. If you agree, can you please merge? Thanks!

**Comment by AnniekStok** [comment]
I updated the figure with the version from #351 for now. I also changed the image data so that the blobs image is not used. @tischi  can you please merge?

**Comment by AnniekStok** [comment]
Was there any specific reason why the code layout for macro and Jython has changed everywhere? I liked it more before, this seems less clear. Sorry if I missed it
image

**Comment by tischi** [comment]
> Was there any specific reason why the code layout for macro and Jython has changed everywhere? 

Yes, see discussion here: https://github.com/NEUBIAS/training-resources/issues/344

Please post there examples where the current rendering does not look nice. When I tested it, it looked good.

**Comment by AnniekStok** [comment]
I see now, I just have a typo in that line so that it is not a proper comment. Fixed that now.

**Pull request merged**